migrations/migrations/20220906155837-user-table.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('users', {
+      id: {
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      first_name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          isAlpha: {
+            msg: 'El nombre solo puede contener letras'
+          },
+          len: {
+            args: [2, 255],
+            msg: 'El nombre tiene que tener como minimo dos caracteres'
+          }
+        }
```

Comment:
I think it is not necessary to put those validations in the migration files, just put the type, allowNull, if the variables ir unique for example.